lib/utils/reify-output.js

```diff
@@ -50,8 +51,12 @@ const reifyOutput = arb => {
     getChildren: d => d.children
   })
 
-  for (const node of actualTree.inventory.filter(n => n.package.funding)) {
-    summary.fund ++
+  if (npm.flatOptions.fund) {
+    const fundingInfo = await getFundingInfo({
```

Comment:
Ohhhhh, I see.  That's why you added `await reifyOutput(arb)` everywhere.  I get it.

If we know we have the tree already, though, couldn't there be a `libnpmfund.readTree(...)` that is synchronous?  Then `libnpmfund.read` just does `if (!tree) { return readTree({...opts, tree: await arb.buidlIdealTree(arbOpts) })` or something?